### PR TITLE
Development

### PR DESCRIPTION
**Pull request by VishnuPrasad-1995**

### CODE REVIEW

File: src/main/java/com/mavericsystems/likeservice/feign/UserFeign.java [hunk 1]
```diff
@@ -6,8 +6,8 @@
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PathVariable;
 
-@FeignClient(name = "user-service",configuration = CustomerRetryClientConfig.class,fallbackFactory = HystrixFallBackFactory.class)
-//@FeignClient(name = "user-service")
+//@FeignClient(name = "user-service",configuration = CustomerRetryClientConfig.class,fallbackFactory = HystrixFallBackFactory.class)
```
Comment (Ramithags) [inline src/main/java/com/mavericsystems/likeservice/feign/UserFeign.java]: remove ghost code

File: src/main/java/com/mavericsystems/likeservice/repo/LikeRepo.java [hunk 2]
```diff
@@ -1,11 +1,14 @@
 package com.mavericsystems.likeservice.repo;
 
 import com.mavericsystems.likeservice.model.Like;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
 import org.springframework.data.mongodb.repository.MongoRepository;
 
 import java.util.List;
 
 public interface LikeRepo extends MongoRepository<Like,String> {
```
Comment (Ramithags) [inline src/main/java/com/mavericsystems/likeservice/repo/LikeRepo.java]: Use @Repository annotation if possible

File: src/main/java/com/mavericsystems/likeservice/service/LikeServiceImpl.java [hunk 3]
```diff
@@ -26,8 +25,60 @@
     @Autowired
```
Comment (Ramithags) [inline src/main/java/com/mavericsystems/likeservice/service/LikeServiceImpl.java:28]: Format project files

**Review by Ramithags — CHANGES_REQUESTED** [review]

**Comment by VishnuPrasad-1995** [comment]
Changes are made as per reviewer, New pull request will be raised with modifications. So this PR is closed

**Comment by VishnuPrasad-1995** [comment]
modification made as per reviewer.

**Pull request closed without merging**